container/worker/run_openqa_worker.sh

```diff
@@ -9,8 +9,15 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+  groupmod -g "$group" --non-unique kvm
+  [[ $(getent group kvm | cut -d ":" -f3) -ne "$group" ]] && { echo "Unable to ensure the ID of the kvm matches the group ID of /dev/kvm. KVM will not work."; exit 1; } 
   usermod -a -G kvm _openqa-worker
+  qemu-kvm -S &
```

Comment:
This is just a symlink to e.g. `qemu-system-x86_64` which you're hard-coding below anyways. So I'd just use `qemu-system-x86_64 -enable-kvm`. You maybe need `-nographic` as well to prevent the GTK3 error in a headless environment.